llvm/test/Transforms/LoopVectorize/check-no-vectorize.ll

```diff
@@ -0,0 +1,35 @@
+; This test asserts that we don't emit both
+; successful and unsuccessful message about vectorization.
+
+; REQUIRES: asserts
+; RUN: opt -passes=loop-vectorize -debug -disable-output < %s 2>&1 | FileCheck %s
+; CHECK-NOT: LV: We can vectorize this loop
+; CHECK: LV: Not vectorizing: Cannot prove legality
+; CHECK-NOT: LV: We can vectorize this loop
+
+@a = global [32000 x i32] zeroinitializer, align 4
+@b = global [32000 x i32] zeroinitializer, align 4
+
+define void @foo() {
+entry:
+  %load_a_gep = load i32, ptr getelementptr inbounds (i8, ptr @a, i64 4), align 4
+  %val_a = load i32, ptr @a, align 4
+  br label %for.body
+
+for.cond.cleanup:                                 ; preds = %for.body
```

Comment:
might be good to move this to the end of the function and shorten name to something like `exit`

Reply:
Done